components/popover_list_members/popover_list_members.jsx

```diff
@@ -93,140 +85,100 @@ export default class PopoverListMembers extends React.Component {
-                if (this.props.currentUserId !== m.id && this.props.channel.type !== Constants.DM_CHANNEl) {
-                    messageIcon = (
-                        <MessageIcon
-                            className='icon icon__message'
-                            aria-hidden='true'
-                        />
-                    );
-                }
+        const isDirectChannel = this.props.channel.type === Constants.DM_CHANNEL;
 
-                let name = '';
-                if (teamMembers[m.username]) {
-                    name = Utils.displayUsernameForUser(teamMembers[m.username]);
-                }
+        const items = this.state.sortedUsers.map((user) => (
+            <PopoverListMembersItem
+                key={user.id}
+                onItemClick={this.handleShowDirectChannel}
+                showMessageIcon={this.props.currentUserId !== user.id && !isDirectChannel}
+                status={user.status}
+                user={user}
+            />
+        ));
 
-                if (name) {
-                    popoverHtml.push(
-                        <div
-                            data-member-id={m.id}
-                            className='more-modal__row'
-                            onClick={this.handleShowDirectChannel}
-                            key={'popover-member-' + i}
-                        >
-                            <ProfilePicture
-                                src={Client4.getProfilePictureUrl(m.id, m.last_picture_update)}
-                                status={m.status}
-                                width='32'
-                                height='32'
-                            />
-                            <div className='more-modal__details'>
-                                <div
-                                    className='more-modal__name'
-                                >
-                                    {name}
-                                </div>
-                            </div>
-                            <div
-                                className='more-modal__actions'
-                            >
-                                {messageIcon}
-                            </div>
-                        </div>
-                    );
-                }
-            });
+        if (this.props.channel.type !== Constants.GM_CHANNEL) {
+            let membersName = (
+                <FormattedMessage
+                    id='members_popover.manageMembers'
+                    defaultMessage='Manage Members'
+                />
+            );
+
+            const manageMembers = canManageMembers(this.props.channel, isChannelAdmin, isTeamAdmin, isSystemAdmin);
+            const isDefaultChannel = ChannelStore.isDefault(this.props.channel);
 
-            if (this.props.channel.type !== Constants.GM_CHANNEL) {
-                let membersName = (
+            if ((manageMembers === false && isDefaultChannel === false) || isDefaultChannel) {
```

Comment:
`!` vs ` === false`?

Reply:
Oh yeah, I meant to change that logic since it also makes no sense because we're checking `!isDefaultChannel || isDefaultChannel` if manageMemebers is false. I'll fix that